logic/scorer/scorer.go

```diff
@@ -87,29 +103,127 @@ func scoreFifteens(ptVals []int) (scoreType, int) {
 	return st, int(numFifteens * 2)
 }
 
-func howManyAddUpTo(goal int, ptVals []int) uint {
-	if len(ptVals) == 0 {
+func howManyAddUpTo(goal int, ptVals values, start int) uint {
+	if start == cardsPerHand {
 		return 0
 	}
 
 	var many uint
-	for i, o := range ptVals {
+	for i := start; i < cardsPerHand; i++ {
+		o := ptVals[i]
 		if o > goal {
-			break
+			continue
 		} else if o == goal {
 			many++
 		} else {
```

Comment:
:cactus: since we're not breaking anymore, I'd almost rather see this as:
```go
if o == goal {
  many++
} else if o < goal {
  // do the subWith
}
```